packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -662,6 +676,15 @@ export class SequenceInterval implements ISerializableInterval {
 		return `${id}`;
 	}
 
+	/**
+	 * {@inheritDoc ISerializableInterval.getRangeLabels}
+	 */
+	public getRangeLabels(): string[] {
```

Comment:
I don't think this PR needs to change the public API of intervals. It looks like we already store the collection label name as an interval prop and correctly strip this at serialization time + populate it at deserialization time. The work item linked is more about making sure application code doesn't modify this value as a call to changeProperties (which it looks like you do have that logic). We should still be able to retrieve it using the same logic we did before.

Reply:
`getIntervalId` is also a useful API that applications need to call though--this one it's not clear that we have any external users that want it, so I'd rather avoid cluttering the API surface unless we get specific, reasonable feedback that it's necessary for some scenario.